snow/consensus/snowball/tree_test.go

```diff
@@ -464,56 +460,6 @@ func TestSnowballCloseTrinary(t *testing.T) {
 	require.False(tree.Finalized())
 }
 
-func TestSnowballAddRejected(t *testing.T) {
```

Comment:
Duplicate of `TestSnowballAddPreviouslyRejected` @StephenButtolph could you confirm this?